modules/aws_asg/main.tf

```diff
@@ -217,6 +217,7 @@ resource "aws_launch_configuration" "workers" {
   name_prefix       = "${var.env}-${var.index}-workers-${var.site}-"
   image_id          = "${var.worker_ami}"
   instance_type     = "${var.worker_instance_type}"
+  key_name          = "aj"
```

Comment:
I think I would feel better about this being an injected variable :grin:, maybe with a default that's an empty string?

Reply:
I had a feeling you'd say that :D
Coming right up.